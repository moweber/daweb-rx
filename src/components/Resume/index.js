import React from "react";
import resume from "../../assets/resume.pdf";

function Resume() {
  return (
    <section className="container mResume">
      <p>
        Click <a href={resume}>here</a> to download a PDF version of my full
        resume.
      </p>
      <h1>Skills</h1>
      <ul className="mResumeSkills">
        <li>Web Development</li>
        <li>Project Management</li>
        <li>Database Management</li>
        <li>Support Management</li>
        <li>Enterprise Content Management</li>
        <li>Application Development</li>
        <li>User / Customer Experience</li>
        <li>User / Staff Training</li>
        <li>Software Testing</li>
        <li>Technical Writing</li>
        <li>Process Workflows</li>
        <li>Data Reporting</li>
      </ul>
      <h1>Software</h1>
      <ul className="mResumeSoftware">
        <li>CSS/HTML</li>
        <li>Java/JavaScript</li>
        <li>SQL Server/MySQL</li>
        <li>UNIX</li>
        <li>Python</li>
        <li>AWS</li>
        <li>PHP</li>
        <li>Android</li>
        <li>PostgreSQL</li>
        <li>C#</li>
        <li>DNN</li>
        <li>SharePoint</li>
        <li>Django</li>
        <li>WordPress</li>
        <li>Power Automate</li>
        <li>REST APIs</li>
        <li>Salesforce</li>
        <li>jQuery/Ajax</li>
        <li>Node.js</li>
        <li>React/JSX</li>
        <li>M365/Office</li>
        <li>JetBrains</li>
        <li>Photoshop</li>
        <li>Illustrator</li>
        <li>Visual Studio</li>
        <li>VS Code</li>
        <li>Power BI</li>
      </ul>
    </section>
  );
}

export default Resume;
