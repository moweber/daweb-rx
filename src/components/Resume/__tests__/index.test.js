import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Resume from "..";

afterEach(cleanup);

describe("Resume component renders", () => {
  it("renders", () => {
    render(<Resume />);
  });

  it("matches snapshot DOM node structure", () => {
    const { asFragment } = render(<Resume />);

    expect(asFragment()).toMatchSnapshot();
  });
});
