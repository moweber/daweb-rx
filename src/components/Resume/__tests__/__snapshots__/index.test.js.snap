// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume component renders matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section
    class="container mResume"
  >
    <p>
      Click 
      <a
        href="resume.pdf"
      >
        here
      </a>
       to download a PDF version of my full resume.
    </p>
    <h1>
      Skills
    </h1>
    <ul
      class="mResumeSkills"
    >
      <li>
        Web Development
      </li>
      <li>
        Project Management
      </li>
      <li>
        Database Management
      </li>
      <li>
        Support Management
      </li>
      <li>
        Enterprise Content Management
      </li>
      <li>
        Application Development
      </li>
      <li>
        User / Customer Experience
      </li>
      <li>
        User / Staff Training
      </li>
      <li>
        Software Testing
      </li>
      <li>
        Technical Writing
      </li>
      <li>
        Process Workflows
      </li>
      <li>
        Data Reporting
      </li>
    </ul>
    <h1>
      Software
    </h1>
    <ul
      class="mResumeSoftware"
    >
      <li>
        CSS/HTML
      </li>
      <li>
        Java/JavaScript
      </li>
      <li>
        SQL Server/MySQL
      </li>
      <li>
        UNIX
      </li>
      <li>
        Python
      </li>
      <li>
        AWS
      </li>
      <li>
        PHP
      </li>
      <li>
        Android
      </li>
      <li>
        PostgreSQL
      </li>
      <li>
        C#
      </li>
      <li>
        DNN
      </li>
      <li>
        SharePoint
      </li>
      <li>
        Django
      </li>
      <li>
        WordPress
      </li>
      <li>
        Power Automate
      </li>
      <li>
        REST APIs
      </li>
      <li>
        Salesforce
      </li>
      <li>
        jQuery/Ajax
      </li>
      <li>
        Node.js
      </li>
      <li>
        React/JSX
      </li>
      <li>
        M365/Office
      </li>
      <li>
        JetBrains
      </li>
      <li>
        Photoshop
      </li>
      <li>
        Illustrator
      </li>
      <li>
        Visual Studio
      </li>
      <li>
        VS Code
      </li>
      <li>
        Power BI
      </li>
    </ul>
  </section>
</DocumentFragment>
`;
